addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4557,6 +4557,7 @@ registry.sizing = SnippetOptionWidget.extend({
             const isGrid = Object.keys(resizeValues).length === 4;
             const isGridHandle = $handle[0].classList.contains('o_grid_handle');
             if (isGrid && !isGridHandle || !isGrid && isGridHandle) {
+                self.options.wysiwyg.odooEditor.automaticStepActive("resizing");
```

Comment:
@robinlej Let's not mix fixes. I don't think this fix is related in any way to the mutex lock. And actually, while I said that the mutex lock can target 17.0 because "risky", this can probably target older versions and deserve its own commit explanation :wink: